packages/server/analysis_pycodes/analysis/publicalerts.py

```diff
@@ -875,6 +891,7 @@ def main(end=datetime.now()):
     query = "SELECT site_id, site_code FROM commons_db.sites WHERE active = 1"
     props = qdb.get_db_dataframe(query)
     # props = props[props.site_code == 'mar']
+    props = props[(props.site_code == 'mar') | (props.site_code == 'umi')]
```

Comment:
kaya ba gawing dynamic to? para incase di tayo nag hhard code ng site sa backend?

Reply:
dynamic naman ito dapat, kung dadalawa lang site sa database, dalawa lang rin irrun nito. Depende kung pano gustong diskarte.

1 site lang irrun niya depende kung sino gumagamit?